### DISCUSSION TITLE
新加坡「Xavier」的AI社會監控論戰

### DISCUSSION THREAD

**Discussion by chun1203** [1101-GS4719-C/Python-Community-GS4719-C#78] [General]
新加坡在今年9月初推出名為「 Xavier」巡邏機器人的測試計劃，執行巡邏警察的任務，負責偵測「不良社會行為」，包括在禁煙區吸煙、違反社交距離、監督非法攤販、或是亂停腳踏車等。
我認為雖然巡邏機器人可以**分擔警察巡邏的工作**，**降低人力短缺的問題**，但是會讓人有無時無刻**被監控**的感覺，也會造成很多**隱私的˙問題**。
[機器戰警巡邏中？新加坡「Xavier」的AI社會監控論戰](https://global.udn.com/global_vision/story/8662/5803237)

**Comment by vvvioletta** [comment]
政府利用科技來提高行政效率已勢不可擋，然而在這樣的趨勢下，如同學所言，會產生許多隱私權的問題。而這時我認為政府應該要制定相應的法律來保護人民，也能透過這個方法讓運用AI這件事更說服人心而較無隱私疑慮。像是近年來，各國包括台灣在積極推動數位身分證的議題一樣，台灣至今進程仍然緩慢，我認為即是相關法律的嚴密程度仍然不足，進而使人民無法信任。倘若今天人民能夠信任政府，那我相信AI絕對會是整體社會進步的強大助力。

**Reply by HSU-YU-JEN** [reply]
同意你的看法~AI的廣泛運用的確是時勢所趨，但同時政府也須加速制定一些相關法律，來遏止隱私權被侵犯的問題
